src/platforms/javascript/common/configuration/install-cdn.mdx

```diff
@@ -0,0 +1,91 @@
+---
+title: CDN Configuration
+sidebar_order: 1
+caseStyle: camelCase
+supportLevel: production
+categories:
+  - browser
+---
+On this page, we provide installation and configuration information for those using a content delivery network (CDN).
+
+<Alert>
+<markdown>
+
+If you don't already have an account and Sentry project established, head over to [sentry.io](https://sentry.io/signup/), then return to this page.
+
+</markdown>
+</Alert>
+
+## CDN Installation
+
+Sentry captures data by using an SDK within your application’s runtime. Add the Sentry SDK to your application:
+
+```html {tabTitle: CDN}
+<script
+  src="https://browser.sentry-cdn.com/{{ packages.version('sentry.javascript.browser') }}/bundle.min.js"
+  integrity="sha384-{{ packages.checksum('sentry.javascript.browser', 'bundle.min.js', 'sha384-base64') }}"
+  crossorigin="anonymous"
+></script>
+```
+<Alert>
+<markdown>
+
+Using `defer`
+
+If you include `defer` in your script your app, to make sure it can monitor everything that follows it. We strongly recommend that if you use `defer`, you both place the script tag for the browser SDK first and also mark it -- and all of your other scripts -- `defer` (but not `async`), guaranteeing that it’s executed before any of the others.
+
+</markdown>
+</Alert>
+
+### Configure and Verify
+
+`init` Sentry as soon as possible in your app, to make sure it can monitor everything that follows it. Once this is done, all unhandled exceptions are automatically captured by Sentry.
+
+The snippet below includes an intentional error, so you can test that everything is working as soon as you set it up. To view and resolve the recorded error, log into [sentry.io](https://sentry.io) and open your project. Clicking on the error's title will open a page where you can see detailed information and mark it as resolved.
+
+```javascript {tabTitle: CDN}
+Sentry.init({ dsn: "___PUBLIC_DSN___" });
+
+myUndefinedFunction(); // To test your setup
+```
```

Comment:
Does this need to be here? It's identical to the version on the main page.

Reply:
Not if it's totally a dupe. I moved it because we had a CDN tab for it - but, to my somewhat untrained eye - this looks the same as what's on the main page.

Reply:
I'll remove.